src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/OrderlyServerAPI.kt

```diff
@@ -43,7 +43,7 @@ class OrderlyServer(private val config: Config,
                     private val httpClient: HttpClient,
                     private val throwOnError: Boolean = false) : OrderlyServerAPI
 {
-    private val urlBase: String = "${config["orderly.server"]}/v1"
+    private val urlBase: String = config["orderly.server"]
```

Comment:
Not all orderly server endpoints begin with `v1` now, so should be added on a case by case basis.